desktop-environment.el

```diff
@@ -182,6 +208,49 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-backlight-percent ()
+  "Return the new keyboard backlight value as a % instead
```

Comment:
Please try to complete the sentence on the first line by simplifying it. You may remove "instead of a single integer" for example.

Reply:
Done.